moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -1009,8 +1072,10 @@ void TrajectoryExecutionManager::execute(const ExecutionCompleteCallback& callba
 
   // start the execution thread
   execution_complete_ = false;
-  execution_thread_ = std::make_unique<boost::thread>(&TrajectoryExecutionManager::executeThread, this, callback,
-                                                      part_callback, auto_clear);
+  execution_thread_ =
+      std::make_unique<boost::thread>([this](const ExecutionCompleteCallback& c, const PathSegmentCompleteCallback& pc,
```

Comment:
```suggestion
      std::make_unique<std::thread>([this](const ExecutionCompleteCallback& c, const PathSegmentCompleteCallback& pc,
```
this way it fits into one line in github diff view :wink:

Reply:
I will replace boost with standard libraries on all occurrences